core/server/data/schema/schema.js

```diff
@@ -450,11 +450,31 @@ module.exports = {
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
     },
+    stripe_products: {
```

Comment:
Not 100% sure why these have moved, it's a little tricky to check that nothing has changed

Reply:
Ah they had originally moved due to a test failure with FK constraint where schema had to be re-ordered to not fail Table creation. I'll push a patch.